tests/series_only/value_counts_test.py

```diff
@@ -0,0 +1,26 @@
+import sys
+from typing import Any
+
+import pytest
+
+import narwhals.stable.v1 as nw
+from tests.utils import compare_dicts
+
+data = {"a": [4, 4, 6, 4, 1, 1]}
+
+
+def test_value_counts(request: Any, constructor: Any) -> None:
+    if "pandas_series_nullable" in str(constructor) or sys.version_info < (
```

Comment:
I think this should be an `and`? It's a bit in an old version of pandas for pandas-nullable series, but the rest of the libraries should run fine on Python 3.8